### PR TITLE
add yzalis ua-parser

### PR DESCRIPTION
**Pull request by mimmi20**
This PR requires PR #80 to be merged before.

### CODE REVIEW

**Comment by mimmi20** [comment]
@jaydiablo Could you review this?

**Comment by jaydiablo** [comment]
This one uses a new "datasource" so I was hoping to be able to address #11 before merging, otherwise the comparison runs aren't accurate.

**Comment by mimmi20** [comment]
ok.

**Comment by jaydiablo** [comment]
This one parses ok, but it fails its own testsuite (mainly some platform/version issues), here are the UAs it fails on:

```
Mozilla/5.0 (Macintosh; U; PPC Mac OS X; sv-se) AppleWebKit/419 (KHTML, like Gecko) Safari/419.3
Mozilla/5.0 (iPhone; CPU iPhone OS 6_0_1 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Version/6.0 Mobile/10A523 Safari/8536.25
Mozilla/5.0 (iPad; CPU OS 5_1 like Mac OS X) AppleWebKit/534.46 (KHTML, like Gecko) Version/5.1 Mobile/9B176 Safari/7534.48.3
Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.2; Win64; x64; Trident/6.0)
Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 6.1; Trident/4.0)
Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.0)
Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.1)
Opera/7.50 (Windows XP; U)
Mozilla/4.0 (compatible; MSIE 5.5; Windows NT 5.0 )
Mozilla/5.0 (Windows; U; Win98; en-US; rv:1.4) Gecko Netscape/7.1 (ax)
Mozilla/2.02E (Win95; U)
Mozilla/4.0 (compatible; MSIE 7.0; Windows Phone OS 7.0; Trident/3.1; IEMobile/7.0; SAMSUNG; OMNIA7)
Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.1; XBLWP7; ZuneWP7)
Mozilla/4.0 (compatible; MSIE 5.5; Windows 98; Win 9x 4.90)
HTC_Touch_3G Mozilla/4.0 (compatible; MSIE 6.0; Windows CE; IEMobile 7.11)
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_6_8) AppleWebKit/537.13+ (KHTML, like Gecko) Version/5.1.7 Safari/534.57.2
Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10_6_7; en-us) AppleWebKit/534.16+ (KHTML, like Gecko) Version/5.0.3 Safari/533.19.4
Mozilla/5.0 (iPhone; U; CPU iPhone OS 4_0 like Mac OS X; en-us) AppleWebKit/532.9 (KHTML, like Gecko) Version/4.0.5 Mobile/8A293 Safari/6531.22.7
Mozilla/5.0 (iPad; U; CPU OS 4_3_5 like Mac OS X; en-us) AppleWebKit/533.17.9 (KHTML, like Gecko) CriOS/23.0.1271.100 Mobile/8L1 Safari/6533.18.5
Mozilla/5.0 (Linux; U; Android 2.3.7; en-us; HTC Desire Build/GRI40; CyanogenMod-7.2.0.1) AppleWebKit/533.1 (KHTML, like Gecko) Version/4.0 Mobile Saf
Mozilla/5.0 (Linux; U; Android 4.0; en-gb; HTC_Desire_C_N/1.45.161.5 Build/IML74K) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Mobile Safari/
Mozilla/5.0 (BlackBerry; U; BlackBerry 9900; en) AppleWebKit/534.11+ (KHTML, like Gecko) Version/7.1.0.346 Mobile Safari/534.11+
BlackBerry9700/5.0.0.862 Profile/MIDP-2.1 Configuration/CLDC-1.1 VendorID/331
BlackBerry 9000: BlackBerry9000/4.6.0.65 Profile/MIDP-2.0 Configuration/CLDC-1.1 VendorID/102
NokiaN-Gage/1.0 SymbianOS/6.1 Series60/1.2 Profile/MIDP-1.0 Configuration/CLDC-1.0
Mozilla/5.0 (SymbianOS/9.4; U; Series60/5.0 Nokia5800d-1b/20.2.014; Profile/MIDP-2.1 Configuration/CLDC-1.1 ) AppleWebKit/413 (KHTML, like Gecko) Safari/413
```

Ideally it would be nice that a parser doesn't fail on its own test suite, usually that implies there's a mapping issue, or perhaps an issue with how the tests are built, or the properties are extracted when parsing.

**Comment by mimmi20** [comment]
I fixed the tests.

File: tests/yzalis/metadata.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "name": "ZSXSoft",
```
Comment (jaydiablo) [inline tests/yzalis/metadata.json]: This needs to be changed to match the test suite provider :)

**Review by jaydiablo — CHANGES_REQUESTED** [review]

File: tests/yzalis/metadata.json (hunk 1 above)
Comment (mimmi20) [reply tests/yzalis/metadata.json]: fixed

**Review by jaydiablo — APPROVED** [review]

**Pull request merged**